index.js

```diff
@@ -1,126 +1,19 @@
 module.exports = {
-  plugins: ['flowtype', 'import', 'prettier', 'promise', 'react', 'react-hooks'],
-  extends: ['plugin:flowtype/recommended', 'plugin:react/recommended', 'airbnb', 'prettier'],
+  plugins: [
+  ],
+  extends: [
+    'plugin:flowtype/recommended', 
+    'plugin:react/recommended', 
+    'airbnb', 
+    'prettier',
+    './rules/prettier.js',
+    './rules/eslint.js',
+    './rules/import.js',
+    './rules/promise.js',
+    './rules/react.js',
+    './rules/react-hooks.js',
+    './rules/flow.js'
```

Comment:
I'd stick to the name of the original plugin/extended config, to make it easier to follow.
```suggestion
    './rules/flowtype.js'
```